src/osdproc.c

```diff
@@ -692,79 +708,155 @@ void draw_time() {
                SIZE_TO_FONT[eeprom_buffer.params.Time_fontsize]);
 }
 
-void draw_CWH(void) {
-  char tmp_str[100] = { 0 };
-  float dstlon, dstlat, dstsqrt;
+// Haversine distance
+// http://www.movable-type.co.uk/scripts/latlong.html
+float get_distance_between_locations_in_meters(float lat_one, 
+                                               float lon_one, 
+                                               float lat_two, 
+                                               float lon_two)
+{
```

Comment:
Maybe rename the parameters to `lat_from, lon_from, lat_to, lon_to`?
It does not matter which way around (I think) but it makes it easier
when reading the function signature because then it is suggested,
that one param goes in one place and the other in the other.